src/js/components/awardv2/idv/activity/ActivityChartTooltip.jsx

```diff
@@ -178,7 +177,8 @@ export default class IdvActivityTooltip extends React.Component {
             (
                 <div>
                     This IDV &#62; {this.getLinks(
-                        `award/${data.parentGeneratedId}`,
+                        `award`,
```

Comment:
can we change this to single quotes instead of backticks since we do not have any data in those strings anymore.  The back ticks are template literals in JS and require an extra step for babel to read and translate them down to regular JS.